scripts/exp_final.py

```
@@ -0,0 +1,114 @@
+            t0 = time.perf_counter(); vm.run("echo hello"); warm.append(time.perf_counter() - t0)
+        rec["warm"] = sum(warm) / len(warm)
+    finally:
+        if vm is not None:
+            try: vm.stop(); vm.delete()
+            except Exception: pass
+    return rec
+
+
+def bench(label, make_vm) -> dict:
+    print(f"\n== {label}: warm-up ==", flush=True)
+    try: cycle(make_vm)
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)